src/graphql/call/fragment.call.graphql

```diff
@@ -33,5 +33,10 @@ fragment call on Call {
     name
     description
   }
+  template {
```

Comment:
I would suggest to not adding values retrieved from resolvers to the fragment and keep fragment lightweight.

Reply:
@jekabs-karklins I see for example `instruments` was added like 9 months ago in this fragment from resolvers. Not sure if I should clean everything up or how do you suggest to do it?

Reply:
I think yes, we should keep the fragments lightweight, so if we can move out more then the better. But I guess not with this PR.